components/script/dom/event.rs

```diff
@@ -216,6 +216,14 @@ impl Event {
         self.target.set(target_);
     }
 
+    pub(crate) fn set_related_target(&self, related_target_: Option<&EventTarget>) {
```

Comment:
I know this is not idiomatic rust, but I don't know how to have both a setter and a getter, without exposing the `MutNullableDom` inside. I don't think we want to expose that field to callers, so I followed the pattern of `set_` and omit `get_` here.

Reply:
That how I understood the idiomatic Rust naming conventions to be honest. `set_field()` and `field()`. 

That said, I think you can omit the underscore here without worry:

```rust
pub(crate) fn set_related_target(&self, related_target: Option<&EventTarget>) {
```

Reply:
Ah wasn't sure, as `target_` also had an underscore. Will remove that and thanks for the clarification on idiomatic Rust.